src/screens/AuthedAppTileScreen.tsx

```diff
@@ -60,8 +69,36 @@ export const AuthedAppTileScreen = ({
     if (activeSubjectId) {
       parsed.patientId = activeSubjectId;
     }
+
+    for (const [key, value] of Object.entries(searchParams)) {
+      parsed[key] = value;
+    }
+
     return `${oauthCallbackUrl}?${queryString.stringify(parsed)}`;
-  }, [account, activeProject, data, oauthCallbackUrl, activeSubjectId]);
+  }, [
+    account,
+    activeProject,
+    data,
+    oauthCallbackUrl,
+    activeSubjectId,
+    searchParams,
+  ]);
+
+  const handlePageLoaded = () => {
+    setIsPageLoaded(true);
+  };
+
+  const dispatchLXFocusEvent = useCallback(() => {
```

Comment:
It would be nice at some point if we could change `lx` to something more meaningful.